src/rf/apps/home/views.py

```diff
@@ -69,6 +69,18 @@ def create_layer(request, username):
     return _save_layer(request, layer, username=username)
 
 
+# Attempts to get key[] out of data
+# if key doesn't exist to handle
+# the way jQuery sends arrays.
+def get_list(key, data):
+    bracket_key = key + '[]'
+    if key in data:
+        return data.getlist(key)
+    elif bracket_key in data:
+        return data.getlist(bracket_key)
+    return []
```

Comment:
Consider extending `QueryDict` and adding this method there. You can wrap `request.POST` / `request.PUT` with a custom `QueryDict` here: https://github.com/azavea/raster-foundry/blob/develop/src/rf/apps/core/decorators.py#L46.